packages/cli/src/commands/verify.ts

```diff
@@ -2,29 +2,60 @@ import pickBy from 'lodash.pickby';
 
 import verify from '../scripts/verify';
 import ConfigVariablesInitializer from '../models/initializer/ConfigVariablesInitializer';
+import Truffle from '../models/initializer/truffle/Truffle';
+import { promptIfNeeded, getContractsList, getNetworkList } from '../utils/prompt';
 
 const name: string = 'verify';
-const signature: string = `${name} <contract-alias>`;
+const signature: string = `${name} [contract-alias]`;
```

Comment:
Keep in mind that if we later add a flag to disable interactive, we may need to keep the old signatures around

Reply:
I was thinking about doing these validations inside the scripts. WDYT?

Reply:
Could work, and it's a good idea since it also checks programmatic usage. Let's see how it goes!